src/raps/actions/swap.ts

```diff
@@ -358,8 +358,7 @@ export const swap = async ({
     ],
     gasLimit,
     hash: swap.hash as TxHash,
-    // TODO: MARK - Replace this once we migrate network => chainId
-    network: ethereumUtils.getNetworkFromChainId(parameters.chainId),
+    network: chainsName[parameters.chainId],
     // chainId: parameters.chainId,
```

Comment:
can also remove line 362 as `chainId` is set on line 318

Reply:
removing, let's see how it goes

Reply:
oh I was referring to the commented out line: `// chainId: parameters.chainId,` that you can remove since the `chainId` is set on line 318 - not sure if you still need `network: chainsName[parameters.chainId],` which you previously had updated